### PR TITLE
bump vector to 0.19.X-alpine

### PR DESCRIPTION
**Pull request by smg247**
We need to mirror the vector image to get around rate-limiting. I created a new `openshiftci` quay.io organization and mirrored the latest minor version of vector: https://quay.io/repository/openshiftci/vector?tab=tags

### CODE REVIEW

**Comment by hongkailiu** [comment]
We have other places using vector.
Could we bump all of them? like, https://github.com/openshift/release/pull/19445/files

**Comment by smg247** [comment]
> We have other places using vector. Could we bump all of them? like, https://github.com/openshift/release/pull/19445/files

Good call. Done.

File: clusters/build-clusters/02_cluster/openshift/api_audit_log/vector-audit-log_daemonset.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ spec:
       dnsPolicy: Default
       containers:
       - name: vector-audit-log
-        image: quay.io/alvaroaleman/vector:0.14.X-alpine
+        image: quay.io/alvaroaleman/vector:0.19.X-alpine
```
[the remark this answers is not in the document]
Comment (smg247) [reply clusters/build-clusters/02_cluster/openshift/api_audit_log/vector-audit-log_daemonset.yaml]: oops...

**Review by hongkailiu — APPROVED** [review]
/hold

Please cancel when you are ready to watch the rolling.

**Pull request merged**